_episodes_rmd/02-raster-plot.Rmd

```diff
@@ -208,65 +207,68 @@ ggplot() +
 
 ## Layering Rasters
 We can layer a raster on top of a hillshade raster for the same area, and use a
-transparency factor to created a 3-dimensional shaded effect. A
+transparency factor to create a 3-dimensional shaded effect. A
 hillshade is a raster that maps the shadows and texture that you would see from
 above when viewing terrain.
 We will add a custom color, making the plot grey. 
 
+First we need to read in our DSM hillshade data and view the structure:
+
 ```{r}
-# import DSM hillshade
 DSM_hill_HARV <-
   raster("data/NEON-DS-Airborne-Remote-Sensing/HARV/DSM/HARV_DSMhill.tif")
 
-# View data structure
 DSM_hill_HARV
 ```
 
+Next we convert it to a dataframe, so that we can plot it using `ggplot2`:
+
 ```{r}
-# convert to a df for plotting
 DSM_hill_HARV_df <- raster::as.data.frame(DSM_hill_HARV, xy = TRUE) 
 
-# View data structure
 str(DSM_hill_HARV_df)
 ```
 
+Now we can plot the hillshade data:
+
 ```{r, fig.width= 7, fig.height=7}
 ggplot() +
  geom_raster(data = DSM_hill_HARV_df , aes(x = x, y = y, alpha = HARV_DSMhill)) + 
     scale_alpha(range =  c(0.15, 0.65), guide = "none")
 ```
 
-
-> ## Data Tip
+> ## Data Tips
 > Turn off, or hide, the legend on a plot using by adding `guide = "none"` 
-to a `scale_something()` function or by setting
-`theme(legend.position="none")`.
+> to a `scale_something()` function or by setting
+> `theme(legend.position="none")`.
+> 
+> The alpha value determines how transparent the colors will be (0 being
+> transparent, 1 being opaque).
 {: .callout}
 
-We can layer another raster on top of our hillshade using by using `add = TRUE`.
-Let's overlay `DSM_HARV` on top of the `hill_HARV`.
+We can layer another raster on top of our hillshade using adding another call to 
+the `geom_raster()` function. Let's overlay `DSM_HARV` on top of the `hill_HARV`.
 
 ``` {r overlay-hillshade, fig.width= 7, fig.height=7}
+
 ggplot() +
     geom_raster(data = DSM_HARV_df , 
                 aes(x = x, y = y, 
                      fill = HARV_dsmCrop)) + 
+  
     geom_raster(data = DSM_hill_HARV_df, 
                 aes(x = x, y = y, 
                   alpha = HARV_DSMhill)) +
+  
     scale_fill_viridis_c() +
+  
     scale_alpha(range = c(0.15, 0.65), guide = "none") +
+  
     ggtitle("Elevation with hillshade") +
     coord_equal()
 
 ```
 
-The alpha value determines how transparent the colors will be (0 being
-transparent, 1 being opaque).
-
-* More information about the Viridis palette used above at
-<a href="https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html" target="_blank">`R` Viridis package documentation</a>.
-
```

Comment:
could move this into lesson one where the comment about viridis being colour-blind friendly sits?

Reply:
i would agree with this